src/shared/utils/index.ts

```diff
@@ -119,15 +126,20 @@ export async function getLists(): Promise<Record<string, Set<string>>> {
   const promises: Array<Promise<void>> = []
   const networks = await getNetworks()
   networks.forEach(async (network) => {
-    promises.push(
-      fetchValidatorList(network.unls[0]).then((blob) => {
-        Object.assign(lists, {
-          [network.id]: blobToValidators(blob),
-        })
-      }),
-    )
+    if (network.unls[0]) {
```

Comment:
instead of hiding it hear return early.  That will unindent more relavant code.

`if(!network.unls[0]){ return }`